src/Kernel32/storebanned/Kernel32.cs

```diff
@@ -3237,8 +3237,10 @@ public static partial class Kernel32
         /// You do not need to close the pseudo-handle when you no longer need it.If you call the <see cref="CloseHandle(IntPtr)"/> function with a
         /// pseudo-handle, the function has no effect.
         /// </remarks>
-        [DllImport(nameof(Kernel32))]
-        public static extern SafeObjectHandle GetCurrentProcessToken();
+        public static SafeObjectHandle GetCurrentProcessToken()
```

Comment:
The 3 apis is not exported in Kernel32.dll, and the original define in `processthreadsapi.h` looks like
```c

#if (_WIN32_WINNT >= _WIN32_WINNT_WIN8)

FORCEINLINE
HANDLE
GetCurrentProcessToken (
    VOID
    )
{
    return (HANDLE)(LONG_PTR) -4;
}

FORCEINLINE
HANDLE
GetCurrentThreadToken (
    VOID
    )
{
    return (HANDLE)(LONG_PTR) -5;
}

FORCEINLINE
HANDLE
GetCurrentThreadEffectiveToken (
    VOID
    )
{
    return (HANDLE)(LONG_PTR) -6;
}

#endif // (_WIN32_WINNT >= _WIN32_WINNT_WIN8)
```